src/components/building/buildingcard.js

```diff
@@ -13,100 +18,111 @@ export default function Buildingcard({
   buildPrice,
   buildPriceTextColor,
   buildYield,
-  buildInProgressTime,
   currentBuildLevel,
   buttonText,
   buttonDisabled,
   buttonFunction,
```

Comment:
Instead of `buttonFunction` you should use an event-like name, e.g. `onActionButtonClick`

Reply:
done. buttonFunction to onActionButtonClick